services/private_ip_manager/src/main/java/com/futurewei/alcor/privateipmanager/repo/IpAddrRangeRepo.java

```diff
@@ -242,11 +252,14 @@ public synchronized IpAddrAlloc allocateIpAddr(IpAddrRequest request) throws Exc
             throw new IpRangeNotFoundException();
         }
 
+        CacheConfiguration cfg = CommonUtil.getCacheConfiguration(getIpAddrCacheName(rangeId));
+        ICache<String, IpAddrAlloc> ipAddrCache = cacheFactory.getCache(IpAddrAlloc.class, cfg);
+
```

Comment:
@DavidLiu506 Please put some comments here to describe why we make these changes.